plugins/identity-shiro/licenses/bcprov-jdk15to18-1.77.jar.sha1

```diff
@@ -0,0 +1 @@
+475984acd55286a49a4acb40c12c3702f72e17ee
```

Comment:
Seems like Identity-shiro's dependency should pivot to use jdk18on too, no?

https://github.com/opensearch-project/OpenSearch/blob/84750b3840b8249ac1363e2e1bfa6b24c174e15f/plugins/identity-shiro/build.gradle#L31

Reply:
Oops must have missed this.